packages/playground/windows/playground-win32/Playground-Win32.cpp

```diff
@@ -59,6 +62,7 @@ struct WindowData {
   bool m_useDirectDebugger{false};
   bool m_breakOnNextLine{false};
   uint16_t m_debuggerPort{defaultDebuggerPort};
+  uint16_t m_theme{0};
```

Comment:
ElementTheme is already an enum, no need to obscure it by using an int here

```suggestion
  xaml::ElementTheme m_theme{xaml::ElementTheme::Default};
```